From f8ddcbd5390e1a6ca2104fec61aa41a625a4e938 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Wed, 4 Dec 2024 15:16:04 +0800
Subject: [PATCH 195/301] AOSCOS: platform: mips: rename dependency for
 LEMOTE3A_LAPTOP

Rename LOONGSON_MACH3X => MACH_LOONGSON64 per commit 6fbde6b492df ("MIPS:
Loongson64: Move files to the top-level directory").

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/platform/mips/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/platform/mips/Kconfig b/drivers/platform/mips/Kconfig
index 59b6027eeadf..7ef9a47556d5 100644
--- a/drivers/platform/mips/Kconfig
+++ b/drivers/platform/mips/Kconfig
@@ -39,7 +39,7 @@ config LS2K_RESET
 
 config LEMOTE3A_LAPTOP
 	tristate "Lemote Loongson-3A Laptop Driver"
-	depends on LOONGSON_MACH3X
+	depends on MACH_LOONGSON64
 	select BACKLIGHT_LCD_SUPPORT
 	select LCD_CLASS_DEVICE
 	select BACKLIGHT_CLASS_DEVICE
-- 
2.48.1

